docs/assembly.rst

```diff
@@ -47,13 +47,20 @@ these curly braces, you can use the following (see the later sections for more d
 
  - literals, i.e. ``0x123``, ``42`` or ``"abc"`` (strings up to 32 characters)
  - opcodes in functional style, e.g. ``add(1, mlod(0))``
- - labels, e.g. ``name:``
  - variable declarations, e.g. ``let x := 7``, ``let x := add(y, 3)`` or ``let x`` (initial value of empty (0) is assigned)
- - identifiers (labels or assembly-local variables and externals if used as inline assembly), e.g. ``jump(name)``, ``3 x add``
- - assignments (in "instruction style"), e.g. ``3 =: x``
- - assignments in functional style, e.g. ``x := add(y, 3)``
+ - identifiers (assembly-local variables and externals if used as inline assembly), e.g. ``add(3, x)``, ``sstore(x_slot, 2)``
+ - assignments, e.g. ``x := add(y, 3)``
  - blocks where local variables are scoped inside, e.g. ``{ let x := 3 { let y := add(x, 1) } }``
 
+The following features are only available for standalone assembly.
+Standalone assembly is supported for backwards compatibility but is not documented
+here anymore.
+
+ - direct stack contral via dup1, swap1, ...
```

Comment:
contral -> control

But I'd format this paragraph a bit more nicer.

Reply:
Any suggestions? Do you want to take this over?

Reply:
Perhaps it's enough to pre-format the instructions using back-ticks. Should we move the note about deprecation of instructions-style assignments up here?